lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -95,11 +97,15 @@ def configure(conf)
         end
       end
 
-      @platform = detect_platform
+      # TODO: Send instance tags and/or hostname as labels as well?
+      @common_labels = {}
+      @running_on_managed_vm = false
```

Comment:
I would leave this before the second "case" so it's closer where it's actually set.